src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -690,14 +691,20 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration = new
 
         protected DynamoViewModel(StartConfiguration startConfiguration)
         {
+            // CurrentDomain_UnhandledException - catches unhandled exceptions that are fatal to the current process. These exceptions cannot be handled and process termination is guaranteed
+            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
```

Comment:
I thought this was initially added to DynamoModel. Can you explain why it has been added here instead of DynamoModel and the same for TaskScheduler.UnobservedException? Since DynamoModel is instantiated before the view model, there could be code that's uncaught by any of these handlers.

Reply:
Will do

Reply:
Ok. Maybe we should decouple the CER crash prompt from DynamoViewModel. Currently, it needs the view model to show the dialog. Would be good to remove the dependency on the view model for this dialog if possible.

Reply:
I am trying that now, but there are too many changes for this PR
I will create a followup